packages/backend/src/models/OrganizationModel.ts

```diff
@@ -237,6 +241,32 @@ export class OrganizationModel {
             })),
         );
 
+        // If default color palette is configured, set it as the active palette
+        if (
+            this.lightdashConfig?.appearance?.defaultColorPalette &&
+            this.lightdashConfig.appearance.defaultColorPalette.length > 0
+        ) {
+            const defaultPaletteName =
+                this.lightdashConfig.appearance.defaultColorPaletteName ||
+                'Default (Custom)';
+
+            const defaultPalette = await this.createColorPalette(
+                org.organization_uuid,
+                {
+                    name: defaultPaletteName,
+                    colors: this.lightdashConfig.appearance.defaultColorPalette,
+                },
+            );
+
+            // Set it as the active palette
+            if (defaultPalette) {
+                await this.setActiveColorPalette(
+                    org.organization_uuid,
+                    defaultPalette.colorPaletteUuid,
+                );
+            }
```

Comment:
I could have appended the default color palette to the batchInsert above, but I'm happy with keeping this logic separate. Let me know if you think differently.